x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/apm_403_response_to_a_post.json

```diff
@@ -12,7 +12,7 @@
   "language": "kuery",
   "license": "Elastic License v2",
   "name": "Web Application Suspicious Activity: POST Request Declined",
-  "query": "http.response.status_code:403 and http.request.method:post",
+  "query": "http.response.status_code:403 and http.request.method:post\n",
```

Comment:
@brokensound77 this was made from `main`.
any idea what this change is about or the version bump?

i think i remember a change with `nested_normalize`?

Reply:
https://github.com/brokensound77/detection-rules/blob/c8be7ef52c921a73e4e9218832fa9cd11c0a0bb9/detection_rules/rule_formatter.py#L51-L52

Reply:
Some change along the way with the refactors was causing queries to flatten (most obvious when running `toml-lint` command). So that was to restore that. As far as trailing newlines, I can't remember what the original was, though based on this, I would assume we stripped trailing newlines.

We should review that formatting logic to be sure it is consistent either way, to prevent unintended version bumps

Reply:
I think what happens is that we want to preserve the original spacing when outputting a rule to TOML, but when generating the hash or dictionary we want to flatten to a single line using our previous logic

Reply:
ah right, that makes sense. Before, the flattening and sorting was done as part of the hashing method. Now it is a shared method. Still not totally sure where the trailing newline diff is at.